src/content-scripts/ElementIndexer.js

```diff
@@ -339,8 +339,16 @@ class ElementIndexer {
 		if(Prefs.Debug_Measure_SearchSpeed)
 			rr = new RateReporter('Found ${Count} Elements in ${Elapsed} (${PerSecond})');
 
-		let docHeight   = document.documentElement.scrollHeight,
-			Buckets     = Prefs.IndexBuckets,
+			let docHeight = document.documentElement.scrollHeight;
```

Comment:
This is all duplicate code and should be put into a function, such as in globals.  Probably name it like GetDocumentDims() : [ int, int ]